{"uuid":"a5211180-ee5e-4e1e-a0b1-5d8d049eb237","historyId":"66e9b83031ffb912b1f05505aceba2b9","status":"failed","statusDetails":{"message":"`cy.request()` failed trying to load:\n\nhttp://localhost:8080/test-utils/reset-database\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: connect ECONNREFUSED 127.0.0.1:8080\n\n-----------------------------------------------------------\n\nThe request we sent was:\n\nMethod: POST\nURL: http://localhost:8080/test-utils/reset-database\n\n-----------------------------------------------------------\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n\nhttps://on.cypress.io/request","trace":"CypressError: `cy.request()` failed trying to load:\n\nhttp://localhost:8080/test-utils/reset-database\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: connect ECONNREFUSED 127.0.0.1:8080\n\n-----------------------------------------------------------\n\nThe request we sent was:\n\nMethod: POST\nURL: http://localhost:8080/test-utils/reset-database\n\n-----------------------------------------------------------\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n\nhttps://on.cypress.io/request\n    at <unknown> (http://localhost:3001/__cypress/runner/cypress_runner.js:132388:70)\n    at tryCatcher (http://localhost:3001/__cypress/runner/cypress_runner.js:1807:23)\n    at <unknown> (http://localhost:3001/__cypress/runner/cypress_runner.js:4203:37)\n    at tryCatcher (http://localhost:3001/__cypress/runner/cypress_runner.js:1807:23)\n    at Promise._settlePromiseFromHandler (http://localhost:3001/__cypress/runner/cypress_runner.js:1519:31)\n    at Promise._settlePromise (http://localhost:3001/__cypress/runner/cypress_runner.js:1576:18)\n    at Promise._settlePromise0 (http://localhost:3001/__cypress/runner/cypress_runner.js:1621:10)\n    at Promise._settlePromises (http://localhost:3001/__cypress/runner/cypress_runner.js:1697:18)\n    at _drainQueueStep (http://localhost:3001/__cypress/runner/cypress_runner.js:2407:12)\n    at _drainQueue (http://localhost:3001/__cypress/runner/cypress_runner.js:2400:9)\n    at Async._drainQueues (http://localhost:3001/__cypress/runner/cypress_runner.js:2416:5)\n    at Async.drainQueues (http://localhost:3001/__cypress/runner/cypress_runner.js:2286:14)\nFrom Your Spec Code:\n    at Context.eval (http://localhost:3001/__cypress/tests?p=cypress/e2e/features/schedule_enrollment.feature:12035:12)\n    at Registry.runHook (http://localhost:3001/__cypress/tests?p=cypress/e2e/features/schedule_enrollment.feature:9264:31)\n    at Object.fn (http://localhost:3001/__cypress/tests?p=cypress/e2e/features/schedule_enrollment.feature:10112:39)\n    at runStepWithLogGroup (http://localhost:3001/__cypress/tests?p=cypress/e2e/features/schedule_enrollment.feature:9860:36)\n    at Context.eval (http://localhost:3001/__cypress/tests?p=cypress/e2e/features/schedule_enrollment.feature:10111:51)\n\nFrom Node.js Internals:\n  RequestError: Error: connect ECONNREFUSED 127.0.0.1:8080\n      at new r (<embedded>:1868:354814)\n      at ee.callback (<embedded>:1868:360815)\n      at e.callback.s.callback [as _callback] (<embedded>:1868:360261)\n      at s._callback.s.callback.s.callback (<embedded>:2086:91494)\n      at ee.emit (node:events:527:28)\n      at ee.onRequestError (<embedded>:2086:100436)\n      at ClientRequest.emit (node:events:539:35)\n      at Socket.socketErrorListener (node:_http_client:454:9)\n      at Socket.emit (node:events:527:28)\n      at emitErrorNT (node:internal/streams/destroy:157:8)\n      at emitErrorCloseNT (node:internal/streams/destroy:122:3)\n      at process.processTicksAndRejections (node:internal/process/task_queues:83:21)"},"stage":"finished","steps":[{"status":"passed","statusDetails":{},"stage":"finished","steps":[{"status":"passed","statusDetails":{},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1746790355734,"name":"log Test starting: Attempt to enroll in a schedule that conflicts with another enrolled schedule at 2025-05-09T11:32:35.733Z","stop":1746790355734}],"attachments":[],"parameters":[],"start":1746790355691,"name":"\"before each\" hook","stop":1746790355734},{"status":"failed","statusDetails":{},"stage":"finished","steps":[{"status":"failed","statusDetails":{},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1746790355738,"name":"POST --- http://localhost:8080/test-utils/reset-database","stop":1746790356441}],"attachments":[],"parameters":[],"start":1746790355737,"name":"Before","stop":1746790356442},{"status":"passed","statusDetails":{},"stage":"finished","steps":[{"status":"passed","statusDetails":{},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1746790356445,"name":"log Test finished: Attempt to enroll in a schedule that conflicts with another enrolled schedule in 711ms","stop":1746790356446},{"status":"passed","statusDetails":{},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1746790356447,"name":"writeFile cypress/reports/test-timing/schedule_enrollment.feature-timing.json","stop":1746790356452},{"status":"passed","statusDetails":{},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1746790356452,"name":"task resetDatabase","stop":1746790361216}],"attachments":[],"parameters":[],"start":1746790356444,"name":"\"after each\" hook","stop":1746790361216},{"status":"passed","statusDetails":{},"stage":"finished","steps":[{"status":"passed","statusDetails":{},"stage":"finished","steps":[{"status":"passed","statusDetails":{},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1746790361217,"name":"task resetDatabase","stop":1746790366008}],"attachments":[],"parameters":[],"start":1746790361217,"name":"resetDatabase","stop":1746790366008}],"attachments":[],"parameters":[],"start":1746790361217,"name":"\"after each\" hook","stop":1746790366008}],"attachments":[{"name":"cypress/e2e/features/schedule_enrollment.feature:2025-05-09T11:32:36.266Z","type":"image/png","source":"40c11eb6-9b7e-481d-a3eb-2c86b4bbd3cc-attachment.png"},{"name":"video recording","type":"video/mp4","source":"f22a89ce-eca0-467b-a6c2-4f76acf4708e-attachment.mp4"}],"parameters":[],"labels":[{"name":"package","value":"cypress.e2e.features.schedule_enrollment.feature"},{"name":"suite","value":"Schedule Enrollment"},{"name":"feature","value":"Schedule Enrollment"}],"links":[],"start":1746790355685,"name":"Attempt to enroll in a schedule that conflicts with another enrolled schedule","fullName":"Attempt to enroll in a schedule that conflicts with another enrolled schedule","stop":1746790356443}